### PR TITLE
API keys: Add capability proto

### PR DESCRIPTION
**Pull request by bduffany**
Figured the proto could contain a list of "positive" capabilities, and if we want to add a distinction between capabilities / restrictions (to avoid DB migrations), then we can hide that gnarliness from the frontend and handle it at the DB layer.

---

**Version bump**: Patch

### CODE REVIEW

File: proto/api_key.proto [hunk 1]
```diff
@@ -16,6 +16,20 @@ message ApiKey {
...
+
+  // A capability associated with an API key.
+  //
+  // Values are powers of 2 so that bitmask operations can be used
+  // to check capabilities.
+  enum Capability {
+    UNKNOWN_CAPABILITY = 0;
+    // Allows the API key to write to the content-addressable store
+    // and action cache.
+    CACHE_WRITE = 1; // 2^0
```
Comment (tylerwilliams) [inline proto/api_key.proto:28]: nit: CACHE_WRITE_CAPABILITY to be consistent with enum naming style

Comment (bduffany) [reply proto/api_key.proto]: Done

**Comment by bduffany** [comment]
Ninja-edited since I realized I forgot to add this to the Update request

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Pull request merged**